pkg/apis/acid.zalan.do/v1/crds.go

```diff
@@ -569,6 +569,9 @@ var PostgresCRDResourceValidation = apiextv1.CustomResourceValidation{
 							},
 						},
 					},
+					"schedulerName": {
+						Type: "string",
+					},
```

Comment:
could you add this part also to the manifests for the Postgresql CRD? [Here](https://github.com/zalando/postgres-operator/blob/master/manifests/postgresql.crd.yaml) and [there](https://github.com/zalando/postgres-operator/blob/master/charts/postgres-operator/crds/postgresqls.yaml).

Reply:
Done.
Also noticed that the custom resource defs follow alphabetical order and I hadn't respected that, so adjusted it as well.